### PR TITLE
Update documentation for adding a tracer and the tutorial

### LINKED ISSUE #489: Create fluff branch/tag/documentation
Create a fluff branch and/or tag that provides a robust implementation of the fluffball tutorial (maybe with some interesting forcing too).  Update the documentation for the tutorial in the Readthedocs.

### PR DESCRIPTION
**Pull request by apcraig**
## PR checklist
- [X] Short (1 sentence) summary of your PR: 
    Update documentation for adding a tracer and the tutorial
- [X] Developer(s): 
    apcraig
- [X] Suggest PR reviewers from list in the column to the right.
- [X] Please copy the PR test results link or provide a summary of testing completed below.
    No testing done, just documentation changes
- How much do the PR code changes differ from the unmodified code? 
    - [X] bit for bit
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on CICE or any other models?
    - [ ] Yes
    - [X] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [X] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/.)
    - [X] Yes
    - [ ] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [ ] No 
- [X] Please document the changes in detail, including _why_ the changes are made.  This will become part of the PR commit log.

Documentation is being update based on the Consortium workshop in 2024.  The adding a tracer section has been updated as has the tutorial section related to adding a fluff tracer.  An example set of code diffs for the tutorial is also includes as reference.

Closes #489

### CODE REVIEW

**Comment by apcraig** [comment]
Two sections of the documentation were modified, 5.5 (Adding tracers) and 6.2 (Icepack Tutorial).  See the readthedocs link above and please suggest further changes.

File: doc/source/appendices/tutorial.rst [hunk 1]
```diff
@@ -23,199 +23,126 @@ Notes:
...
+* Port the model and verify you can build and run the model
 
-* The workflow guide is oriented toward setting up CICE rather than Icepack, but the same workflow applies to Icepack standalone.  Icepack can be set up and run as an independent model following the same workflow.
+You may be able to leverage the conda port to build and run the model, you may be working on a supported machine, or you may have to port the model to your machine.  Porting, setting up cases, building, running, and running test suites are all documented in the Icepack User Guide, https://cice-consortium-icepack.readthedocs.io/en/latest/index.html.  There are many ways to setup and run the model.
 
+**Note:** The workflow guide is oriented toward setting up CICE rather than Icepack, but the same workflow applies to Icepack standalone.
 
-Clone Icepack
--------------------
+To summarize the steps in greater details **assuming use of conda in a Mac or Linux environment**.
```
Comment (eclare108213) [inline doc/source/appendices/tutorial.rst:42]: Note that these steps only need to be done once.
Remove the s from 'details'.

File: doc/source/appendices/tutorial.rst [hunk 2]
```diff
@@ -23,199 +23,126 @@ Notes:
...
-* What happens to ice area and ice thickness around October 1, 2015? Why do you see this signal?
-* How does your output compare to the sample output provided for this release? (hint: see the wiki!)
-
-Take a step back and think about all the directories and files you have created. The Icepack “sandbox” was cloned from GitHub and has the actual Icepack code.
-
-* There is a particular case directory for building and launching the code, and some output (e.g. job log) are copied.
-* There is a particular run directory for each case. This is where the model is run and big files are found.
-
-
-Set Up a Longer Run
```
Comment (eclare108213) [inline doc/source/appendices/tutorial.rst:148]: I think setting up a longer run and modifying a physics option in namelist or a code parameter are useful parts of the tutorial separate from adding a tracer.  I recommend making section 6.2.3 be something like "Modify the configuration or code" and section 6.2.4 "Add a new tracer"

**Review by eclare108213 — COMMENTED** [review]
This is looking good.  I like the extra fluff.diff file.  My main comment is that some of the tutorial exercises that aren't specifically tracer related should be pulled into their own section.

File: doc/source/appendices/tutorial.rst [hunk 3]
```diff
@@ -112,26 +112,15 @@ Look at the output.  Go to the ICE_RUNDIR (defined in **icepack.settings**). A s
...
-
-* Add some constant atmospheric forcing and review results
-
-* Change the dependency to ice volume, how do the results change?
-
-* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
+Modify the Configuration or Code
+------------------------------------
 
 * Set Up a Longer Run.  Modify npt in icepack_in.  npt defines the number of timesteps to run.  Details about namelist options are in the documentation (:ref:`case_settings`).
```
Comment (eclare108213) [inline doc/source/appendices/tutorial.rst:118]: Remove the capitalization (or add it in the next bullet for consistency).
Format npt as a code variable.

File: doc/source/appendices/tutorial.rst [hunk 4]
```diff
@@ -145,4 +134,20 @@ fluff tracer.  Then update the timeseries plotting script to plot the fluffballs
...
+
+* First, set the initial value, physics, sources, and sinks of fluff to zero and make sure fluff values remain zero throughout the run
+
+* Add some constant atmospheric forcing and review results
+
+* Change the dependency to ice volume, how do the results change?
+
+* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
+
+**NOTE:** The file, **doc/source/tutorial/fluff.diff** in the Icepack repository, demonstrates code differences for this fluffball activity as implemented in a version of Icepack from July, 2024.  These code difference may not be directly applicable to other code versions, but these diffs provide an example of the typical code differences required to add the tracer, fluff.
```
Comment (eclare108213) [inline doc/source/appendices/tutorial.rst:153]: In the main documentation (section 5.5), add a note that there is a tutorial exercise on adding tracers in the appendix?  The tutorial doesn't have a lot of information except for the fluff.diff file, which someone who hasn't looked at the tutorial might find helpful.

File: doc/source/appendices/tutorial.rst (hunk 1 above)
Comment (apcraig) [reply doc/source/appendices/tutorial.rst]: I added some words in a few places about "done once per user" or "done once per machine".

File: doc/source/appendices/tutorial.rst (hunk 2 above)
Comment (apcraig) [reply doc/source/appendices/tutorial.rst:148]: I reorganized this as suggested.

File: doc/source/appendices/tutorial.rst (hunk 3 above)
Comment (apcraig) [reply doc/source/appendices/tutorial.rst]: Done.

File: doc/source/appendices/tutorial.rst (hunk 4 above)
Comment (apcraig) [reply doc/source/appendices/tutorial.rst:155]: Done.

File: doc/source/appendices/tutorial.rst [hunk 5]
```diff
@@ -23,199 +23,133 @@ Notes:
...
-Revert your code changes::
+* Change the dependency to ice volume, how do the results change?
 
-  cd ~/Icepack
-  git status
-  git checkout columnphysics/icepack_mechred.F90
-  git status
+* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
 
+**NOTE:** The file, **doc/source/tutorial/fluff.diff** in the Icepack repository, demonstrates code differences for this fluffball activity as implemented in a version of Icepack from July, 2024.  These code difference may not be directly applicable to other code versions, but these diffs provide an example of the typical code differences required to add the tracer, fluff.
```
Comment (eclare108213) [inline doc/source/appendices/tutorial.rst:155]: very minor corrections:
"These code differences..."
"example of the typical code modifications required..."

**Review by eclare108213 — APPROVED** [review]

**Comment by eclare108213** [comment]
@dabail10 please take a look at this when you have a chance.  Thanks!

**Review by dabail10 — APPROVED** [review]

**Pull request merged**